app/views/categories/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="navigation">
+  <ul>
+    <li><%= link_to image_tag('backarrow.png', alt: 'Back'), root_path, class: 'nav-link' %></li>
+    <h2>Categories</h2>
+    <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" %></li>
+  </ul>
+</div>
+
+<div class="categories-list">
+  <% @categories.each do |category| %>
+    <%= link_to category do %>
+      <div class="category-item">
+        <div class="category-left">
+          <% if category.icon.present? %>
+            <%= image_tag category.icon, alt: category.name, class: 'category-icon' %>
+          <% end %>
+          <h2><%= category.name %></h2>
+        </div>
+        <p>Total Amount: <%= category.purchases.sum(:amount) %></p>
+      </div>
+    <% end %>
+  <% end %>
+</div>
+<div class="new-category">
+  <div class="new-category-inner">
+    <%= link_to "Add a New Category", new_category_path, class: "new-category-button" %>
+  </div>
+</div>
```

Comment:
- [ ] For the `Category#Index` page, the **_Add a New Category_** button should be positioned at the bottom of the page based on the Behance design. Currently, the button is not designed that way. Kindly redesign the button so that your application looks identical to that of the Behance design. 😎 
- [ ] Also, on the page you need to redesign the navigation bar to look identical to that on the Behance template provided. 👍 
- [ ] Please note the spacing around elements for each card and work on it. The amount shouldn't be underlined because it isn't on the template (In my opinion I don't think you nee the _**Total Amount**_ text.🖱️ 

| Your Design  | Behance Design |
| ------------- | ------------- |
| ![image](https://github.com/misalazeem/Budget-App/assets/77106962/72c87a3a-c369-42b6-a504-d0a128a8ee7c) | ![image](https://github.com/yegonkimutai/budget-app/assets/77106962/7514c6b7-46db-41fe-ad8d-7a4902203586)  |